static/templates/settings/realm-filter-settings-admin.handlebars

```diff
@@ -23,7 +23,7 @@
             </li>
         </ul>
 
-        <table class="table table-condensed table-striped admin_filters_table">
+        <table class="table table-condensed admin_filters_table">
```

Comment:
Why this change?  Should be a separate commit regardless, but I think this looks nicer this way here:￼

![image](https://user-images.githubusercontent.com/2746074/43611113-f9612674-965c-11e8-946b-25f9cb87573a.png)

Reply:
Yeah, it was overriding style `margin-top: 20px` so removed that class.

Reply:
Cool, a few thoughts on this:
(1) You should read about CSS precedence; there's a better way to ensure the CSS gets used.  
(2) Generally, this sort of refactor to another component should go in its own commit, both to make it obvious you're changing another component, and also so there's a place for the explanation for why this is needed (i.e what you just sent).